vars/workers.groovy

```diff
@@ -65,6 +65,9 @@ def base(Map params, Closure closure) {
 
       dir("kibana") {
         checkoutInfo = getCheckoutInfo()
+        if (buildState.add('checkoutInfo', checkoutInfo)) {
```

Comment:
nit: There are a small number of possible return values for `buildState.add()`, but I think this would be easier to understand at first glance as:

```suggestion
        // use the `checkoutInfo` as a flag to indicate that we've already reported the pending commit status
        if (!buildState.has('checkoutInfo')) {
          buildState.set('checkoutInfo', checkoutInfo)
```